package-lock.json

```diff
@@ -11501,28 +11501,28 @@
 					"dependencies": {
 						"abbrev": {
 							"version": "1.1.1",
-							"resolved": false,
+							"resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
@Automattic/team-calypso is there anything to be worried about with all these package-lock changes? Don't want to mess anything else up when merging!